subprojects/language-groovy/src/main/java/org/gradle/api/tasks/compile/GroovyCompile.java

```diff
@@ -383,4 +385,14 @@ public void setGroovyClasspath(FileCollection groovyClasspath) {
     protected JavaToolChainFactory getJavaToolChainFactory() {
         throw new UnsupportedOperationException();
     }
+
+    /**
+     * Injects and returns an instance of {@link FeaturePreviews}.
+     *
+     * @since 5.6
```

Comment:
I think the current convention is to not add Javadoc and `@since`/`@Incubating` to methods which are there for injection.